code/src/components/Title.js

```diff
@@ -0,0 +1,38 @@
+import React from "react"
+import "./title.css"
+
+export const Title = (props) => {
+
+  const title = props.album.name
+  const artists = props.album.artists
+  const href = props.album.external_urls.spotify
+  const artistUrl = props.album.artists[0].external_urls.spotify
```

Comment:
This gives the same url for all artists. Try mapping over the artists' array instead

Reply:
We don't really know what you mean here, because when you click on the artist the specific url for the artist works with the solution we got here. Can you please explain a bit more why we can't do it like we have done it! :)

Reply:
Yes of course! If you click the second or the third artist on an album that has more than one artist you will be sent to the first artist anyway because you hardcoded the index of the artist's URL: `[0]`